modules/ztp-pre-hub-install-factory-pipeline-checks.adoc

```diff
@@ -0,0 +1,105 @@
+baremetal                                   4.9.13   True        False         False      178m
+cloud-controller-manager                    4.9.13   True        False         False      3h
+cloud-credential                            4.9.13   True        False         False      179m
+cluster-autoscaler                          4.9.13   True        False         False      178m
+config-operator                             4.9.13   True        False         False      3h
+console                                     4.9.13   True        False         False      168m
+csi-snapshot-controller                     4.9.13   True        False         False      178m
+dns                                         4.9.13   True        True          False      178m
+etcd                                        4.9.13   True        False         False      177m
+image-registry                              4.9.13   True        False         False      172m
+ingress                                     4.9.13   True        False         False      173m
+insights                                    4.9.13   True        False         False      172m
+kube-apiserver                              4.9.13   True        False         False      175m
+kube-controller-manager                     4.9.13   True        False         False      176m
+kube-scheduler                              4.9.13   True        False         False      175m
+kube-storage-version-migrator               4.9.13   True        False         False      179m
+machine-api                                 4.9.13   True        False         False      175m
+machine-approver                            4.9.13   True        False         False      179m
+machine-config                              4.9.13   False       False         True       102m
+marketplace                                 4.9.13   True        False         False      178m
+monitoring                                  4.9.13   False       True          True       93m
+network                                     4.9.13   True        True          True       3h
+node-tuning                                 4.9.13   True        False         False      178m
+openshift-apiserver                         4.9.13   True        False         False      173m
+openshift-controller-manager                4.9.13   True        False         False      174m
+openshift-samples                           4.9.13   True        False         False      172m
+operator-lifecycle-manager                  4.9.13   True        False         False      179m
+operator-lifecycle-manager-catalog          4.9.13   True        False         False      178m
+operator-lifecycle-manager-packageserver    4.9.13   True        False         False      173m
+service-ca                                  4.9.13   True        False         False      179m
+storage                                     4.9.13   True        True          False      179m
+----
```

Comment:
The cluster is still Progressing with the installation + you have here some degraded components 
should be 
`True False False" for all operators